VultisigApp/VultisigApp/Model/CoinMeta.swift

```diff
@@ -7,8 +7,10 @@
 
 import Foundation
 import WalletCore
+import SwiftData
 
-struct CoinMeta : Hashable,Codable{
+@Model
+class CoinMeta : Hashable, Codable {
```

Comment:
Can we please don't save `CoinMeta` into swiftdata , the reason we add CoinMeta is because we have too much crash issues with SwiftData and Coin ,  

If we do this , we will get the issue again